go/exorelay/exorelay_test.go

```diff
@@ -126,6 +164,56 @@ func FeatureContext(s *godog.Suite) {
 		}
 		return nil
 	})
+
+	s.Step(`^I register a handler for the "([^"]*)" message$`, func(messageName string) error {
+		mapping := map[string]func(map[string]interface{}){}
+		mapping[messageName] = func(payload map[string]interface{}) {
+			handlerCalled = true
+			savedPayload = payload
+		}
+		exoInstance.BindMessageHandlers(mapping)
```

Comment:
@alexdavid showed me an example and we couldn't see any reason why the user wouldn't supply them all at once. 

If we are specifying one by one, then we either need error checking that you can only specify one handler per message or need to implement an event emitter.